Webex Connect Flows/README.md

```diff
@@ -49,7 +49,8 @@ In 3.1 version of workflows, the flows are enhanced with Security scan results t
 
 ## Version History
 
-| Workflows version | High level Description                      |
-| ----------------- | ------------------------------------------- |
-| 3.1               | Simplified flows with Security scan results |
-| 3.0               | Simplified flows                            |
\ No newline at end of file
+| Workflows version | High level Description                                |
+|-------------------|-------------------------------------------------------|
+| 3.2               | Simplified flows with Survey and Set Contact Priority |
+| 3.1               | Simplified flows with Security scan results           |
+| 3.0               | Simplified flows                                      |
```

Comment:
3.0 -> The 2.x flows undergone a major update in 3.0 where the contact center logic in contact resolve, routed, modified, closed usecases are simplified with new nodes. 

3.1 -> Malware scanning feature is integrated to all the attachments send in all channels. The channel start nodes are enhanced to have new set of variables to identify the malware and security scan results.

3.2 -> Flow nodes are enhanced to configure contact priority and allow system messages like Survey links, Thank you messages gets delivered to Customer.

Can you please check and rephrase accordingly.

Reply:
Sure